buildkernel

```diff
@@ -1540,8 +1540,19 @@ ensure_efi_partition_mounted() {
     check_key_file_if_on_efi_system_partition
 }
 enter_build_directory() {
-    show "Proceeding - entering ${LINUXDIR}..."
-    cd "${LINUXDIR}"
+    if [ -z ${LINUXBUILDDIR+x} ]; then
+        LINUXBUILDDIR=$LINUXDIR
+    else
+        if [ -d "$LINUXBUILDDIR" ]; then
+            show "Proceeding - Cleaning ${LINUXBUILDDIR}..."
+            rm -r ${LINUXBUILDDIR}
+        fi
+        show "Proceeding - copying from ${LINUXDIR} to ${LINUXBUILDDIR}..."
+        cp -LR ${LINUXDIR} ${LINUXBUILDDIR}
```

Comment:
I choose it to make sure to actually copy `LINUXDIR`, which itself is a symbolic link and not to just copy the symbolic link. Given that there are relative symbolic links within the kernel source it is indeed suboptimal.
A smarter way to copy the target of `${LINUXDIR}` and not the symbolic link itself would be to use `${LINUXDIR}/` I think.